lib/sentry/envelope.ex

```diff
@@ -126,4 +154,15 @@ defmodule Sentry.Envelope do
         throw(error)
     end
   end
+
+  defp item_to_binary(json_library, %Transaction{} = transaction) do
+    case transaction |> Sentry.Client.render_transaction() |> json_library.encode() do
+      {:ok, encoded_transaction} ->
+        header = ~s({"type": "transaction", "length": #{byte_size(encoded_transaction)}})
```

Comment:
Less bytes for free 🙃 

```suggestion
        header = ~s({"type":"transaction","length":#{byte_size(encoded_transaction)}})
```